dbt/main.py

```diff
@@ -328,6 +330,20 @@ def parse_args(args):
             should be a YAML string, eg. '{my_variable: my_value}'"""
     )
 
+    # if set, log all cache events. This is extremely verbose!
+    base_subparser.add_argument(
+        '--log-cache-events',
+        action='store_true',
+        help=argparse.SUPPRESS,
+    )
```

Comment:
Or maybe some sort of `--log=dbt.cache`, where `log` is a repeatable argument that enables log propagation for the given package? Usually when I want more granular logging, one thing I don't want is more granular logging everywhere.

Reply:
Hmmm maybe @drewbanin would have something more useful to say here. My specific concern is having proliferating set of flags related to logging specific event types. Do you have a use case in mind for `--log=dbt.cache`? Is that for ease of development?

Reply:
Yeah. This logging is really only useful for debugging narrow cache-related issues. I think `--log=dbt.cache` and stuff like it would probably have to be a whole new PR, the way we currently set up logging doesn't really play nice with that structure.

Reply:
Yeah, the cache logging here is _way_ too verbose to be useful in the default case. I like the idea of turning on logs per-module, but that seems more useful for developers of dbt than users of dbt itself. We could also make it a `config` in `profiles.yml`  I suppose? Like:

```
config:
  logging: 
    modules: ['dbt.cache', 'dbt.whatever']
    ...
```

I imagine there's other logging things to configure too. Regardless, don't know that we need to implement it in this PR. 

I agree that `--log-cache-events` feels weird, but since it's set to "suppressed", I feel great about removing it in the future if we do something more comprehensive around logging.

Reply:
Yeah. I don't think anyone should ever be passing `--log-cache-events` in production, unless maybe we ask someone to do so as part of tracking down a cache consistency issue. It's nice to have it for integration tests though, I've already tracked down an intermittent cache bug due to the extra output.